packages/@sanity/block-tools/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@sanity/block-tools",
+  "version": "0.115.2",
+  "description": "Can format HTML, Slate JSON or Sanity block array into any other format.",
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "compile": "babel src --copy-files --out-dir lib",
+    "compile:watch": "npm run compile -- --watch",
+    "prepublish": "in-publish && npm run compile && sanity-check || not-in-publish",
+    "lint": "eslint src --quiet",
+    "test": "NODE_ENV=test jest",
+    "coverage": "NODE_ENV=test jest --coverage"
+  },
+  "jest": {
+    "testEnvironment": "node",
+    "setupTestFrameworkScriptFile": "./test/setup.js",
+    "collectCoverageFrom": [
+      "src/**.js"
+    ]
+  },
+  "keywords": [],
+  "author": "Sanity.io <hello@sanity.io>",
+  "license": "MIT",
+  "dependencies": {
+    "get-random-values": "^1.2.0",
+    "lodash": "^4.17.4"
+  },
+  "devDependencies": {
+    "@sanity/schema": "^0.113.7",
+    "assert": "^1.4.1",
+    "babel-cli": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-plugin-css-modules-transform": "^1.1.0",
+    "babel-plugin-lodash": "^3.2.11",
+    "babel-plugin-syntax-class-properties": "^6.13.0",
+    "babel-plugin-syntax-flow": "^6.18.0",
+    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-flow-comments": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "eslint": "^4.6.1",
+    "eslint-config-sanity": "^3.1.0",
+    "eslint-import-resolver-webpack": "^0.8.3",
+    "eslint-plugin-flowtype": "^2.35.1",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-react": "^7.4.0-rc.1",
+    "flow-bin": "^0.54.1",
+    "flow-typed": "^2.0.0",
+    "in-publish": "^2.0.0",
+    "jest": "^21.2.1",
+    "jest-cli": "^21.2.1",
+    "jsdom": "11.3.0",
+    "xpath": "^0.0.24"
+  },
+  "peerDependencies": {},
+  "directories": {
+    "test": "test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sanity-io/sanity.git"
+  },
+  "bugs": {
+    "url": "https://github.com/sanity-io/sanity/issues"
+  },
+  "homepage": "https://github.com/sanity-io/sanity#readme"
```

Comment:
Should be `https://www.sanity.io`, see https://github.com/sanity-io/sanity/blob/master/scripts/normalizePackageFields.js